pkg/cmd/tests/options.go

```diff
@@ -118,6 +130,11 @@ func (o *TestFrameworkOptions) AddFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVarP(&o.ObjectStorageBucket, "object-storage-bucket", "", o.ObjectStorageBucket, "Name of the object storage bucket.")
 	cmd.PersistentFlags().StringVarP(&o.GCSServiceAccountKeyPath, "gcs-service-account-key-path", "", o.GCSServiceAccountKeyPath, "Path to a file containing a GCS service account key.")
 	cmd.PersistentFlags().StringVarP(&o.S3CredentialsFilePath, "s3-credentials-file-path", "", o.S3CredentialsFilePath, "Path to the AWS credentials file providing access to the S3 bucket.")
+	cmd.PersistentFlags().StringVarP(&o.ScyllaDBVersion, "scylla-db-version", "", o.ScyllaDBVersion, "Version of ScyllaDB to use in tests.")
+	cmd.PersistentFlags().StringVarP(&o.ScyllaManagerVersion, "scylla-manager-version", "", o.ScyllaManagerVersion, "Version of Scylla Manager to use in tests.")
+	cmd.PersistentFlags().StringVarP(&o.ScyllaManagerAgentVersion, "scylla-manager-agent-version", "", o.ScyllaManagerAgentVersion, "Version of Scylla Manager Agent to use in tests.")
+	cmd.PersistentFlags().StringVarP(&o.ScyllaDBUpdateFrom, "scylla-update-from", "", o.ScyllaDBUpdateFrom, "Version of Scylla Update to use in tests.")
```

Comment:
s/scylla-update-from/scylladb-update-from-version

Reply:
also `Version of Scylla Update to use in tests.` doesn't make much sense to me (applies to upgrade as well), how about "Version of ScyllaDB to update from"?

Reply:
still applies partially, s/scylla/scylladb